mlflow/langchain/__init__.py

```diff
@@ -151,6 +158,13 @@ def save_model(
 
                      .. Note:: Experimental: This parameter may change or be removed in a future
                                              release without warning.
+    :param loader_fn: A function that takes a string `persist_dir` defined below as the argument
+                      and returns a retriever. This is required for RetrievalQA models.
+    :param persist_dir: The directory where the retriever is stored. The `loader_fn` takes this
+                        string as the argument to load the retriever. This is required for
+                        RetrievalQA models. Note that MLflow does not manage the contents of this
+                        directory, so it is up to the user to ensure that the directory is properly
+                        populated.
```

Comment:
Let's say I log a model with this argument and want to serve it on databricks model serving. What do I need to do to make it work?